dev/com.ibm.ws.anno/src/com/ibm/ws/annocache/targets/cache/internal/TargetCacheImpl_ReaderBinary.java

```diff
@@ -241,12 +247,20 @@ public void requireHeader(String tableName, String tableVersion) throws IOExcept
                     " for [ " + getPath() + "]");
         }
 
-        if ( !tableVersion.equals(actualVersion) ) {
+        String matchVersion = null;
+        for ( String tableVersion : tableVersions ) {
+            if ( actualVersion.equals(tableVersion) ) {
+                matchVersion = tableVersion;
+                break;
+            }
+        }
+        if ( matchVersion == null ) {
             throw new IOException(
                 "Unexpected table version [ " + actualVersion + " ]:" +
-                " Expecting table version [ " + tableVersion + " ]" +
+                " Expecting table versions [ " + tableVersions + " ]" +
```

Comment:
Again toString of `tableVersions` will not be very useful.  Surround with Array.toString()?

Reply:
Fixed.